packages/@react-spectrum/list/src/InsertionIndicator.tsx

```diff
@@ -1,30 +1,25 @@
 import {classNames} from '@react-spectrum/utils';
 import {ItemDropTarget} from '@react-types/shared';
-import listStyles from './listview.css';
+import listStyles from './styles.css';
 import {ListViewContext} from './ListView';
 import React, {useContext, useRef} from 'react';
 import {useVisuallyHidden} from '@react-aria/visually-hidden';
 
 interface InsertionIndicatorProps {
-  target: ItemDropTarget
+  target: ItemDropTarget,
+  isPresentationOnly?: boolean
 }
 
 export default function InsertionIndicator(props: InsertionIndicatorProps) {
   let {dropState, dropHooks} = useContext(ListViewContext);
-  const {target} = props;
+  const {target, isPresentationOnly} = props;
+
   let ref = useRef();
   let {dropIndicatorProps} = dropHooks.useDropIndicator(props, dropState, ref);
   let {visuallyHiddenProps} = useVisuallyHidden();
 
   let isDropTarget = dropState.isDropTarget(target);
 
-  // If aria-hidden, we are either not in a drag session or the drop target is invalid.
-  // In that case, there's no need to render anything at all unless we need to show the indicator visually.
-  // This can happen when dragging using the native DnD API as opposed to keyboard dragging.
-  if (!isDropTarget && dropIndicatorProps['aria-hidden']) {
-    return null;
-  }
-
```

Comment:
I see this was added in the commit to account for the shifts in layout. Do we need to change this logic or was the absolute position css change to the ListView InsertionIndicator not sufficient? I guess there isn't a strong reason not to keep the row div rendered though...

Reply:
If I leave this bit in, I get a layout shift only on the last row. And it seemed better to keep rendering the row IMO